conf/sqs_conf.py

```diff
@@ -0,0 +1,12 @@
+"""
+Queue conf
+"""
+from botocore.config import Config
```

Comment:
Nice touch using botocore.config. I had not come across this before. Akkul found `decouple` which is also pretty cool for reading config.